bindings/xml/comment-xml-schema.rng

```diff
@@ -959,6 +959,13 @@
           <param name="pattern">.*\S.*</param>
         </data>
       </element>
+      <element name="Attribute">
```

Comment:
Sorry I missed this. It looks like you forgot to update the schema and XML printing.